src/gtc_unstructured/cpp_util/unstructured/tests/regression/fvm_nabla/nabla_cuda.hpp

```diff
@@ -26,16 +27,23 @@ namespace gridtools::usid::cuda::nabla_impl_ {
     struct kernel_1 {
         GT_FUNCTION auto operator()() const {
             return [](auto &&ptr, auto &&strides, auto &&neighbors) {
-                field<pnabla_MXX_tag>(ptr) = sum_neighbors<double, v2e_tag>(
-                    [](auto &&p, auto &&n) { return field<zavgS_MXX_tag>(n) * field<sign_tag>(p); },
-                    ptr,
-                    strides,
-                    neighbors);
-                field<pnabla_MYY_tag>(ptr) = sum_neighbors<double, v2e_tag>(
-                    [](auto &&p, auto &&n) { return field<zavgS_MYY_tag>(n) * field<sign_tag>(p); },
+                field<pnabla_MXX_tag>(ptr) = 0.0;
+                field<pnabla_MYY_tag>(ptr) = 0.0;
```

Comment:
it's a pessimization to use field accessor as an accumulator.
I would suggest to use local variable for that pattern as you do in `kernel_0`.